performance-tests/unwrapped_doc.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I count 8 fields. let's add more to reach 20 fields. I don't care much about what's in the field just make it not too long for now.

Could you give me the result on your machine for the following:
- 1 000 5
- 10 000 5
- 50 000 5

Once done, update the document to change one field to contain a base64 image and repeat the test